shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx

```diff
@@ -545,11 +568,18 @@ const AddModal = (props: IProps) => {
                   : null}
               </div>
 
-              <Button className="mt-4" size="small" type="primary" onClick={addItem}>
+              <Button ghost className="mt-4" size="small" type="primary" onClick={addItem}>
                 {i18n.t('common:add')}
               </Button>
               <h4 className="mt-4 mb-3 text-sm">{i18n.t('msp:number of retries')}</h4>
-              <InputNumber value={retry} min={0} max={10} defaultValue={2} onChange={(v) => updater.retry(v)} />
+              <InputNumber
+                parser={(formatRetry: string) => (/^\d+$/.test(formatRetry) ? formatRetry : formatRetry[0])}
```

Comment:
what does this mean? keep no float?  try `precision ` instead

Reply:
ok,modified